pallets/remote-asset-manager/src/lib.rs

```diff
@@ -98,5 +98,16 @@ pub mod pallet {
         ) -> DispatchResult {
             todo!()
         }
+
+        fn bond(_asset: <T as Config>::AssetId, _amount: <T as Config>::Balance) -> DispatchResult {
+            todo!()
```

Comment:
Guess these `todos!()`s make the CI sad

Reply:
Makes sense for now, would also make testing easier